packages/edit-site/src/components/editor/index.js

```diff
@@ -260,17 +259,36 @@ export default function EditSiteEditor( { isPostsList = false } ) {
 											showTooltip
 											tooltipPosition="middle right"
 											onClick={ () => {
-												setCanvasMode( 'view' );
 												// TODO: this is a temporary solution to navigate to the posts list if we are
 												// come here through `posts list` and are in focus mode editing a template, template part etc..
 												if (
 													isPostsList &&
 													params?.focusMode
 												) {
-													history.push( {
-														page: 'gutenberg-posts-dashboard',
-														postType: 'post',
-													} );
+													history.push(
+														{
+															page: 'gutenberg-posts-dashboard',
+															postType: 'post',
+														},
+														undefined,
+														{
+															transition:
+																'canvas-mode-view-transition',
+														}
+													);
+												} else {
+													history.push(
+														{
+															...params,
+															canvasMode:
+																undefined,
+														},
```

Comment:
If I don't remove the `canvas` param from the URL like this:

```suggestion
															canvas: undefined,
														},
```

Clicking the WP logo when editing a template or pattern returns me to this page:

<img width="1558" alt="Screenshot 2024-10-18 at 12 08 09 pm" src="https://github.com/user-attachments/assets/e94a52bd-2323-455b-a0fe-8f313327061e">

I don't know if that's intentional or not.